lib/cssom.js

```diff
@@ -11,12 +11,26 @@
 declare class StyleSheet {
   disabled: boolean;
   href: string;
+  media: MediaList;
   ownerNode: Node;
   parentStyleSheet: ?StyleSheet;
   title: string;
   type: string;
 }
 
+declare class StyleSheetList {
+  length: number;
+  [index: number]: StyleSheet;
+}
+
+declare class MediaList {
+  mediaText: string;
+  length: number;
+  item(index: number): string;
```

Comment:
I've also accounted for the fact that the braces return undefined if the index is not found.

```javascript
document.querySelector("link[rel=stylesheet]").media = "screen";
document.styleSheets[0].media.item(1)
// null
document.styleSheets[0].media[1]
//  undefined
```